src/components/tooltip/stories/tooltip.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import { Container, HStack, Stack } from '@chakra-ui/layout';
+import { PhoneIcon, SearchIcon } from '@chakra-ui/icons';
+import { Button } from '../../button';
+import { Tooltip } from '../index';
```

Comment:
index is not needed.
```suggestion
import { Tooltip } from '../';
```